src/main/java/uk/gov/hmcts/reform/hmc/helper/hmi/HmiSubmitHearingRequestMapper.java

```diff
@@ -36,4 +38,22 @@ public HmiSubmitHearingRequest mapRequest(Long hearingId, HearingRequest hearing
             .build();
     }
 
+    public HmiSubmitHearingRequest mapRequest(Long hearingId, UpdateHearingRequest hearingRequest) {
```

Comment:
Should be able to remove this method completely and avoid the duplication if we have `versionNumber` as an argument in the existing `mapRequest()`.